src/components/PrivateRoute.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { Navigate } from "react-router-dom";
+
+
+const PrivateRoute = ({ authenticated, children }) => {
+
```

Comment:
In App.jsx wordt die useContext enkel gebruikt om aan dit component een prop door te geven. Dit component had prima dit zelf kunnen doen
```suggestion
const PrivateRoute = ({ children }) => {
  const { isAuth: authenticated } = useContext(AuthContext);
```